### ISSUE TITLE
Better navigation in directory tree

### ISSUE THREAD

**Issue by orschiro** [prose/prose#749]
Hi there,

I am essentially missing a button to go one level up in the directory tree.

Let's say I am editing an `.md` file and want to get back to my `_posts` directory to edit another one, then there is currently no good way of doing this. Could we think of adding a button to navigate up the directory?

Such a button could ideally be placed somewhere in the header:

![img004](https://cloud.githubusercontent.com/assets/170947/4101262/0f5f3aee-30de-11e4-9ed4-d2020118a4ab.png)

Thanks for listening!

**Comment by jmatsushita** [comment]
Hi there, I was searching for an existing issue which are about the fact that there is currently only a way to navigate to the root of a repo from the editor view. I think this issue reflects this best and in that sense is not really a duplicate of the other issues mentioned before which are about preventing to navigate away from the editor with unsaved changes.

The sidebar idea could address this possibly, another, maybe simpler way would be to display breadcrumbs similar to the browsing view (`../_posts/static`)

![image](https://cloud.githubusercontent.com/assets/356097/15021893/7cf93644-1229-11e6-85db-369e8664ac56.png)

And display it next to the link back to the repo at the top of the editor view (`prose/starter`)

![image](https://cloud.githubusercontent.com/assets/356097/15021907/90bf972c-1229-11e6-808c-04e2c6191bba.png)

This could be for instance `prose/starter > ../_posts/static` and maybe even `prose/starter > ../_posts/static/0100-01-01-about.md`?

cc @dereklieu

**Comment by dereklieu** [comment]
Yeah, I find myself wishing I could go back easier without going to the root. I think breadcrumbs could be a nice way to do this.

**Comment by huksley** [comment]
I am having the same problem navigating, especially when making small changes to number of files.

I found using BACKSPACE works going to _posts folder.
